client/codegen.yml

```diff
@@ -1,5 +1,9 @@
 overwrite: true
-schema: ["https://demo-open.msupply.org/graphql", "./packages/common/src/additions.schema.graphql"]
```

Comment:
Probable best to leave this one as is, I made a note in this [issue](https://github.com/openmsupply/open-msupply/issues/688) that we should probably do export schema and point this file to it

Reply:
[648ecaa](https://github.com/openmsupply/open-msupply/pull/693/commits/648ecaa127ce3875bb799344d7e6f29ebd4524e9) missed it ;x